Gruntfile.js

```diff
@@ -164,7 +173,7 @@ module.exports = ( grunt ) => {
 					'css-dev/**/*.scss',
 					'css-dev/**/*.hbs',
 				],
-				tasks: [ 'kss' ],
+				//tasks: [ 'kss' ],
```

Comment:
You trying to tell me something? Did I over complicate something here? ;)

Reply:
Ha! Honestly, something is broken with KSS now and I have no idea how to get it fixed. I think it's something with the KSS theme. There's been something wonky with all the themes using it, so I'm disabling it here for now.